src/components/DonationForm/MultiStepDonation.vue

```diff
@@ -33,11 +33,13 @@ interface Props {
 	stepControllers: StepController[];
 	pageScroller?: PageScroller;
 	formActionOverride?: string;
+	submitCallback?: () => void;
```

Comment:
For the sake of our future selves we should probably comment this. Somthing like "The component triggers this callback before it submits the form data to the donation page. You can use it for additional event tracking, but you should not use it for validation."

Reply:
You actually reminded me that I want to put it into the 2025 cleanup doc because I think it should be removed after the soft close test.

If the new fee thingy becomes standard (which I suspect it will) then we should check for it and fire that event in the donation form.